src/main/scala/outwatch/dom/helpers/Builder.scala

```diff
@@ -46,17 +47,17 @@ final class DynamicAttributeBuilder[T](parts: List[String]) extends Dynamic with
 }
 
 object KeyBuilder {
-  def :=(key: Key.Value) = IO.pure(Key(key))
+  def :=(key: Key.Value): IO[Key] = IO.pure(Key(key))
 }
 
 object ChildStreamReceiverBuilder {
-  def <--[T](valueStream: Observable[T])(implicit conv: T => VNode): IO[ChildStreamReceiver] = {
-    IO.pure(ChildStreamReceiver(valueStream.map(conv)))
+  def <--[T](valueStream: Observable[T])(implicit r: StaticVNodeRender[T]): IO[ChildStreamReceiver] = IO {
+    ChildStreamReceiver(valueStream.map(v => r.render(v).unsafeRunSync()))
```

Comment:
Why do you run the `child`/`children` IOs here and not before the Snabbdom patch?